examples/model_building/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """
-Cropped Decoding on BCIC IV 2a Dataset
-======================================
+Cropped Decoding on BCIC IV 2a Dataset [1]_
+===========================================
```

Comment:
```suggestion
Cropped Decoding on BCIC IV 2a Dataset
======================================
```
I think it is better to avoid links in title.